app-backup/snapper/snapper-0.5.4.ebuild

```diff
@@ -22,8 +22,7 @@ RDEPEND="dev-libs/boost:=[threads]
 	sys-apps/util-linux
 	sys-libs/zlib
 	virtual/libintl
-	btrfs? ( >=sys-fs/btrfs-progs-3.17.1 )
-	ext4? ( sys-fs/e2fsprogs )
+	>=sys-fs/btrfs-progs-3.17.1
```

Comment:
Isn't there a use case for `USE="-btrfs lvm"`? ;-)

Reply:
it does not build without btrfs-progs installed even if you disable the switch, so that would have to be fixed to preserve the btrfs use flag.

Reply:
Then it makes sense to me.